kits/arm/ex_impedance_control.py

```diff
@@ -54,11 +51,26 @@
 if enable_logging:
     arm.group.start_log('dir', 'logs', mkdirs=True)
 
-print('Commanded gravity-compensated zero force to the arm.')
-print('  b2 - Toggles an impedance controller on/off:')
-print('          ON  - Apply controller based on current position')
-print('          OFF - Go back to gravity-compensated mode')
-print('  b1 - Exits the demo.')
+# Print Instructions
+instructions = """Commanded gravity-compensated zero force to the arm.
+B1 - Exit Demo
+B2 - Toggles an impedance controller on/off:
+          ON  - Apply controller based on current position
+          OFF - Go back to gravity-compensated mode
+"""
+print(instructions)
```

Comment:
Again, IMO we should try to consistently set instructions on terminal, device, or both.

Reply:
This applies to all of these examples -- I'll stop commenting on each one separately :)